polkadot-parachains/src/chain_spec.rs

```diff
@@ -318,6 +320,7 @@ pub fn statemint_local_config(id: ParaId) -> StatemintChainSpec {
 // Not used for syncing, but just to determine the genesis values set for the upgrade from shell.
 pub fn statemint_config(id: ParaId) -> StatemintChainSpec {
 	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("ss58Format".into(), 0.into());
```

Comment:
Great! should we add the equivalent on here: https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/src/chain_spec.rs#L355

I suspect for statemine we should add something like:

`properties.insert("ss58Format".into(), 2.into());`

Reply:
Maybe in this PR we might solve the same issue for both chains. What do you think?

Reply:
That's the dev config, we usually just use the default `42` for that

Reply:
Ok! this is fine, i would suggest to keep the same config for everything ( with the  ss58Format on the properties or not)